### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by majorshatz**
Here's some changes to the JS. I can't figure out how to get the info from the token yet or from the image input to be inputted into the item.
Also Jquery script is there but I don't think it's working. Not showing up as loading in console log. Will look into later. I have to work on Big Data and go to class.

### CODE REVIEW

File: templates/Register.html [hunk 1]
```diff
@@ -23,22 +26,114 @@
...
+    "time();calendar(); spotifytoken(); googletoken(); maps(); pin(); weather();twittertoken(); name();email(); facepath(); postIt(); printC()"-->
+    <input onclick= "test()" class="btn btn-lg btn-success" type="submit">
     <input class ="btn btn-lg btn-danger" type="button" onclick="window.location.href='{{url_for('index')}}'" value="Cancel" />
 </form>
 
 <form action="{{url_for('logout')}}">
 
 </form>
 </body>
+<!--not sure how to call token-->
```
Comment (tenoriojuann) [inline templates/Register.html]: what do you ben how to call token?

File: templates/Register.html [hunk 2]
```diff
@@ -23,22 +26,114 @@
...
+}
+}
+function calendar(){
+    if(document.getElementById('calendar').checked) {
+        userChoices.calendar=true;
+    } else {
+        userChoices.calendar=false;
+}
+}
+function spotifytoken(){
```
Comment (tenoriojuann) [inline templates/Register.html]: remember these are flags now not tokens. You can check the naming convention on the DB to match this.

File: templates/Register.html [hunk 3]
```diff
@@ -23,22 +26,114 @@
...
+    <input onclick= "test()" class="btn btn-lg btn-success" type="submit">
     <input class ="btn btn-lg btn-danger" type="button" onclick="window.location.href='{{url_for('index')}}'" value="Cancel" />
 </form>
 
 <form action="{{url_for('logout')}}">
 
 </form>
 </body>
+<!--not sure how to call token-->
+<script>
```
Comment (tenoriojuann) [inline templates/Register.html]: Put this JS code in the JavaScript folder.

File: templates/Register.html [hunk 4]
```diff
@@ -23,22 +26,114 @@
         <input name="spotifytoken" id="spotify" type="checkbox">
     </p>
 
+
     <Label for="pin">4 character pin: </Label>
     <input name="pin" type="text" Placeholder="****" required pattern=".{4,4}" maxlength="4" minlength="4">
     <br>
     <p>Picture of face:</p>
-    <input name="facepath"  type="file" accept="image/*" capture="camera" required>
+    <input name="facepath" id="facepath"  type="file" accept="image/*" capture="camera" required>
```
Comment (tenoriojuann) [inline templates/Register.html]: You do not need to ask for an image or path. Josh and I will be writing the script to get the facial recognition stuff working and we will save it directly to the DB.

File: templates/Register.html [hunk 5]
```diff
@@ -23,22 +26,114 @@
...
 <form action="{{url_for('logout')}}">
 
 </form>
 </body>
+<!--not sure how to call token-->
+<script>
+    var userChoices={
+        name:"name" ,
+        email:"rando@gmail.com",
+        googletoken:"temp",
```
Comment (tenoriojuann) [inline templates/Register.html]: no more google token

**Comment by tenoriojuann** [comment]
Also, Add a function to send the userPreferences object to the /register POST endpoint

File: templates/Register.html (hunk 3 above)
Comment (tenoriojuann) [reply templates/Register.html]: in its own file

File: templates/Register.html (hunk 4 above)
Comment (majorshatz) [reply templates/Register.html]: In the write ups we did. The front end was supposed to pull an image from the phone(camera/storage) or device they were using on the front end. This prompts for a camera/ file that can be stored.

File: templates/Register.html (hunk 1 above)
Comment (majorshatz) [reply templates/Register.html]: I'm not sure how to get information once they login with the google account. The post is supposed to contain the googletoken, email, and username. I haven't figured out how to get that information yet so I can store it in a js object that I can then post with all the other info.

File: templates/Register.html (hunk 3 above)
Comment (majorshatz) [reply templates/Register.html]: I thought you said we couldn't do relative linking? Or is js different than css?

File: templates/Register.html (hunk 2 above)
Comment (majorshatz) [reply templates/Register.html]: Yea, they weren't changed at the time of creation so I haven't changed them yet.

File: templates/Register.html (hunk 3 above)
Comment (tenoriojuann) [reply templates/Register.html]: you cant do relative linking but you can do this  {{url_for('logout')}}

File: templates/Register.html (hunk 4 above)
Comment (tenoriojuann) [reply templates/Register.html]: The thing is, the api we will be using for this is in Python, so it makes more sense to just have the mirror take the picture. Besides there is no way to activate the camera on the phone from a web page

Comment (majorshatz) [reply templates/Register.html]: The script there enables capture of images with the a phone. I haven't been able to test it yet, I'll take another look online after class. At one of the meeting we talked about not having a physical button that the mirror uses and just getting the image through the registration page. If I misunderstood that is fine but I'm pretty sure our documentation shows us getting it on the registration page.

File: templates/Register.html (hunk 1 above)
Comment (majorshatz) [reply templates/Register.html]: I think I just don't understand how logging on with the google login works.  That method may work, we may need to talk about it later.

File: templates/Register.html (hunk 4 above)
Comment (tenoriojuann) [reply templates/Register.html]: It lets you upload an image but to capture one.

it should be triggered after we send the preferences and pin to the back end. The facial recognition script should start up and then save the image.

File: templates/Register.html (hunk 1 above)
Comment (tenoriojuann) [reply templates/Register.html]: either that or see if you can call {{currentUser}} as it is a variable that exists currenlty

**Comment by tenoriojuann** [comment]
@dsundby10 Isnt the registration page the one you just did a lot of work on?

@majorshatz check with @dsundby10

**Comment by majorshatz** [comment]
To answer @dsundby10 question, yea the registration page is the one I've been doing js and html for for the most part. I add a post function for the preferences and two more functions as well. I plan to also move the script to it's own folder.

File: templates/Register.html (hunk 4 above)
Comment (majorshatz) [reply templates/Register.html]: Alright I can change that in the next commit. I won't be back home for awhile so I can't currently change it.

**Comment by tenoriojuann** [comment]
So since @dsundby10 did a bunch of stuff on this, what do you want to do with this PR @majorshatz ??

**Comment by majorshatz** [comment]
I don't know what my options are; I had some js I wanted to add and he added css is that the issue? Is my js not in his version? If that's the case I can redo my portion. I have it saved and it wouldn't take to long.
If there is a way to push both changes we could do that, but if not my portion wouldn't take to long to redo.

**Comment by tenoriojuann** [comment]
Is all he has CSS changes? and you have the JS? if that's the case then just update this PR, if not you should talk to him about how to proceed.

He did say that there was going to be a PR from him later today.

**Pull request merged**